src/app/courses/course/status-shadow.directive.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { Component } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { StatusShadowDirective } from './status-shadow.directive';
+
+@Component({
+  template: `
+  <h2 [agmpStatusShadow]="'04/25/2020'">Upcoming. With green shadow</h2>
```

Comment:
The same as above, there are no negative scenarios.

Reply:
done